app/services/current-session.js

```diff
@@ -70,4 +71,8 @@ export default class CurrentSessionService extends Service {
   get isImpersonator() {
     return isPresent(this.impersonation.role);
   }
+
+  get isKabinetDossierBeheerder() {
+    return this.role.uri === CONSTANTS.USER_ROLES.KABINET_DOSSIERBEHEERDER;
```

Comment:
Dit gaat geen rekening houden met impersonation. @ValenberghsSven misschien is dit waar je het over had, dat de impersonation hier niet werkt? Not sure.

In any case, als een admin gebruiker een ander profiel impersoneert staat de rol die we willen checken in de `impersonation` service. Ik moet wel toegeven dat wat we hier exposen niet mega duidelijk is, maar je zou het bijvoorbeeld zo kunnen aanpakken:
```js
get isKabinetDossierBeheerder() {
  return (this.impersonation.role?.uri ?? this.role.uri) === CONSTANTS.USER_ROLES.KABINET_DOSSIERBEHEERDER;
}
```
Hiermee check je eerst of een impersonated role bestaat, dan gebruik je om te vergelijken. Als de impersonated role niet bestaat ga je vergelijken met de gewone user role.

Reply:
Ik ga met een permission proberen werken ipv deze check.
met een permission komen we normaal in dezelfde role terecht.